src/VirtoCommerce.WebHooksModule.Data/Services/WebHookManager.cs

```diff
@@ -61,7 +62,7 @@ public virtual async Task<int> NotifyAsync(WebhookRequest request, CancellationT
             {
                 // TechDebt: Here we could create a lot of tasks. Need to add throttling. Also need to decrease pool threads usage.
                 tasks.AddRange(request.WebHooks.Skip(i).Take(_webhooksPerButch)
-                        .Select(x => Task.Run(() => NotifyWebHook(request.EventId, request.EventObject, x, cancellationToken)))
+                        .Select(x => Task.Run(() => NotifyWebHook(request.EventId, request.WebhooksPayload?.FirstOrDefault(r => r.Key.EqualsInvariant(x.Id)).Value ?? string.Empty, x, cancellationToken)))
```

Comment:
Why not passing a null? To catch real value from this code we need to use `string.IsNullOrWhitespace` condition, but `string.IsNullOrEmpty` is preferable when we working with non-user data. Consider to
```suggestion
                        .Select(x => Task.Run(() => NotifyWebHook(request.EventId, request.WebhooksPayload?.FirstOrDefault(r => r.Key.EqualsInvariant(x.Id)).Value, x, cancellationToken)))
```